### PR TITLE
feature: add Chinese stake pool

### PR DESCRIPTION
**Pull request by detailyang**

### CODE REVIEW

**Comment by dajohi** [comment]
@detailyang please rebase.  now written in Go.

File: index.php [hunk 1]
```diff
@@ -99,6 +99,14 @@
         "LastUpdated" => 0,
         "Network" => "mainnet",
         "URL" => "https://ultrapool.eu",
+    ),
+    "China" => array(
```
Comment (jolan) [inline index.php]: The identifier is just the next letter of the NATO phonetic alphabet.  Please use "Oscar" instead of "China".

Comment (detailyang) [reply index.php]: @jolan Many thanks before this i dont know NATO phonetic alphabet:(

**Comment by detailyang** [comment]
@jolan Done. I push a new PR at #7

**Pull request closed without merging**